Neos.ContentRepository/Classes/Command/NodeCommandControllerPlugin.php

```diff
@@ -1362,14 +1312,40 @@ protected function fixShadowNodesInWorkspace(Workspace $workspace, $dryRun, Node
                 if ($nodeDataOnSamePath !== null) {
                     continue;
                 }
-
-                if (!$dryRun) {
-                    $nodeData->createShadow($nodeDataSeenFromParentWorkspace->getPath());
-                }
-                $fixedShadowNodes++;
+                $newShadowNodes[] = ['nodeData' => $nodeData, 'shadowPath' => $nodeDataSeenFromParentWorkspace->getPath()];
             }
         }
 
-        return $fixedShadowNodes;
+        return $newShadowNodes;
+    }
+
+    /**
+     * Attaches a new event handler
+     *
+     * @param string $eventIdentifier one of the EVENT_* constants
+     * @param \Closure $callback a closure to be invoked when the corresponding event was triggered
+     * @return void
+     */
+    public function on($eventIdentifier, \Closure $callback)
+    {
+        $this->eventCallbacks[$eventIdentifier][] = $callback;
+    }
+
+    /**
+     * Trigger a custom event
+     *
+     * @param string $eventIdentifier one of the EVENT_* constants
+     * @param array $eventPayload optional arguments to be passed to the handler closure
```

Comment:
```suggestion
     * @param mixed[] $eventPayload optional arguments to be passed to the handler closure
```

Reply:
`array` and `mixed[]` are pretty much interchangeable, no?